optaplanner-persistence/optaplanner-persistence-jpa/pom.xml

```diff
@@ -171,6 +165,18 @@
       </testResource>
     </testResources>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <configuration>
+          <ignoredUsedUndeclaredDependencies>
+            <dependency>org.jboss.spec.javax.transaction:jboss-transaction-api_1.2_spec</dependency>
+          </ignoredUsedUndeclaredDependencies>
+          <ignoredUnusedDeclaredDependencies>
+            <dependency>jakarta.transaction:jakarta.transaction-api</dependency>
+          </ignoredUnusedDeclaredDependencies>
```

Comment:
I'm assuming that this is a runtime dependency and the plugin simply doesn't see that we will crash without it?

Reply:
See my reply to your previous comment; once we solve it, this extra configuration might go.